app/src/main/java/net/bible/android/control/search/SearchControl.kt

```diff
@@ -229,6 +250,7 @@ class SearchControl @Inject constructor(
         get() = isSearchShowingScripture || !currentDocumentContainsNonScripture()
 
     companion object {
+        lateinit var originalSearchString: String
```

Comment:
This feels a bit unstable. It would be better to do this without this, feels hack.